### PR TITLE
Cleaned up style

### PR DESCRIPTION
**Pull request by lucbettaieb**
Ran linter and cleaned up a bunch of style stuff within all of open_karto including my additions.

### CODE REVIEW

File: include/open_karto/Karto.h [hunk 1]
```diff
@@ -5521,7 +5571,7 @@ namespace karto
      */
     virtual kt_bool IsFree(const Vector2<kt_int32s>& rPose) const
     {
-      kt_int8u* pOffsets = (kt_int8u*)GetDataPointer(rPose);
+      kt_int8u* pOffsets = reinterpret_cast<kt_int8u*>(GetDataPointer(rPose));  // Changed to a reinterpret_cast
```
Comment (mikeferguson) [inline include/open_karto/Karto.h]: we can remove this comment -- the history will be enough

File: src/Mapper.cpp [hunk 2]
```diff
@@ -2072,14 +2185,14 @@ namespace karto
     m_Initialized = false;
   }
 
-  kt_bool Mapper::Process(Object* /*pObject*/)
+  kt_bool Mapper::Process(Object*  /*pObject*/)  // TODO(lucbettaieb)  What even is this doing?
```
Comment (mikeferguson) [inline src/Mapper.cpp]: The parameter is required to match the function signature -- but if it had a name, then we would get an unused variable warning. This syntax works around both issues. (We can remove the comment)

**Comment by mikeferguson** [comment]
@lucbettaieb two comments to remove, and then squash, and we can merge (the devel job is now working, and has cleared the farm)

**Comment by lucbettaieb** [comment]
@mikeferguson Got those changed.  Should be good to go.

**Pull request merged**